libphobos/libdruntime/core/sys/posix/stdio.d

```diff
@@ -279,7 +279,7 @@ unittest
 { /* Note: open_memstream is only useful for writing */
     import core.stdc.string : memcmp;
     char* ptr = null;
-    char[] testdata = ['h', 'e', 'l', 'l', 'o', 0];
+    char[6] testdata = ['h', 'e', 'l', 'l', 'o', 0];
```

Comment:
I'll have a look if there's a notable difference in library size. For executables: When using shared libs this doesn't matter and the linker strips modules when linking with static libs, so it shouldn't really be a problem.